test/framework/debug.go

```diff
@@ -255,9 +255,21 @@ func DebugKube(cluster Cluster, mesh string, namespaces ...string) {
 	Logf("saving export of cluster %q for mesh %q to a file %q", cluster.Name(), mesh, kumaExportPath)
 	Expect(os.WriteFile(kumaExportPath, []byte(out), 0o600)).To(Succeed())
 
+	configDump(kumactlOpts, debugPath, cluster, mesh, "dataplane")
+	configDump(kumactlOpts, debugPath, cluster, mesh, "zoneegress")
+}
+
+func configDump(kumactlOpts kumactl.KumactlOptions, debugPath string, cluster Cluster, mesh string, dpType string) {
```

Comment:
Can dpType be an enum of sort? you know to avoid the usual `zoneegress` vs `zone-egress` annoyance :)